src/Illuminate/Notifications/NotificationSender.php

```diff
@@ -171,7 +171,7 @@ protected function shouldSendNotification($notifiable, $notification, $channel)
      * Queue the given notification instances.
      *
      * @param  mixed  $notifiables
-     * @param  array[\Illuminate\Notifications\Channels\Notification]  $notification
+     * @param  \Illuminate\Notifications\Notification  $notification
```

Comment:
The implementation does not expects an array on this parameter

Reply:
Hi @williamdes 

Maybe this one you want to take a look on how doctum is handling, the error message was odd for this line:

~~~text
ERROR: The "r" @param tag variable name is wrong (should be "notification") on "Illuminate\Notifications\NotificationSender::queueNotification" in build/doctum/laravel/src/Illuminate/Notifications/NotificationSender.php:177
~~~

As you can see the `@param` variable was not called `r`. Maybe it is because it used this odd syntax with brackets

Reply:
Ha, thanks for pointing this out
I had some chance this time because you looked to the old errors report, please have a look to the "5.3.0-dev" one instead :)

Reply:
Now I see, thanks for the heads up